leaf-ui/js/displayAnalysis.js

```diff
@@ -320,12 +313,36 @@ function addAnalysisConfig() {
     $(".result-elements").css("background-color", "");
     var buttonLabel = currAnalysisConfig.id + "-button";
     var label = currAnalysisConfig.id + "-dropdown";
-    $("#analysis-sidebar").append(
-        '<button class="log-elements" id="'+currAnalysisConfig.id+'" style="padding: 12px; font-size: 16px; border: none; outline: none; background-color:#A9A9A9">' + currAnalysisConfig.id + '</button><div style="position:absolute; display:inline-block"><button class="dropdown" id= "'+buttonLabel+'" style="padding: 12px; font-size: 16px; height: 42px; border: none; outline: none;"><i class="fa fa-caret-down fa-2x" style="cursor:pointer;"></i></button>'
+    // Add a larger div to contain the configuration "<config id>-container"
+    $("#configurations").append(
+        '<div class = "analysis-configuration" id="' + currAnalysisConfig.id + '-container">'
+        + '<button class="log-elements" id="'+currAnalysisConfig.id+'" style="padding: 12px; font-size: 16px; border: none; outline: none; background-color:#A9A9A9">' + currAnalysisConfig.id + '</button><div style="position:absolute; display:inline-block"><button class="dropdown" id= "'+buttonLabel+'" style="padding: 12px; font-size: 16px; height: 42px; border: none; outline: none;"><i class="fa fa-caret-down fa-2x" style="cursor:pointer;"></i></button>'
         + '</div><div class = "dropdown-container" id="'+label+'"></div>'
+        + '</div>'
       );
 }
 
+/**
+ * Removes an analysis configuration from the analysisMap and UI sidebar
+ * TODO implement with a button
+ */
+function removeConfiguration() {
```

Comment:
If I'm not mistaken, this function was part of the initial approach to clearing the analysis config sidebar for loading a new model, but isn't needed with the current fix. I do think we'll need a removeConfiguration() eventually, but I agree that it could be useful to be able to pass a config id

Reply:
Yes so I was going to use this func before and ended up not needing it, but since we are planning on a delete functionality I left it there. I will update it to pass a config id in once I get the feature implemented! I was previously thinking that on a click event to delete you should be looking at the config you want to delete, so the currConfig will be set to the one you want to delete. Didn't fully implement the feature on this branch because I didn't want to tangle too much up into this pull request.